src/main/webapp/iaResults.jsp

```diff
@@ -2119,11 +2119,19 @@ $(document).ready(function(){
 });
 
 function isProjectSelected() {
-	let dropdownVal = $("#projectDropdown").val();
-	if (dropdownVal&&dropdownVal!=""&&dropdownVal!="null"&&dropdownVal!=NONE_SELECTED) {
+	<%
+	if(request.getParameter("projectIdPrefix")!=null && !request.getParameter("projectIdPrefix").trim().equals("")&& !request.getParameter("projectIdPrefix").trim().equals("None Selected")){
+	%>
```

Comment:
Let's use NONE_SELECTED. Ideally, even a prop to avoid english bias. But, obv. not a hill I will die on.